### PR TITLE
add example of isSequenceOf relation to Audio and Video

### PR DESCRIPTION
**Pull request by BalduinLandolt**

### CODE REVIEW

File: rosetta.xml [hunk 1]
```diff
@@ -1515,6 +1515,20 @@
         </resptr-prop>
     </resource>
 
+    <!-- :MinimalisticImagePartOfABook -->
+    <resource label="Minimalistic ImagePartOfABook" restype=":MinimalisticImagePartOfABook" id="minimalistic_image_part_of_a_book_obj_0" permissions="res-default">
```
Comment (irinaschubert) [inline rosetta.xml:1519]:
```suggestion
    <resource label="Minimalistic Image" restype=":MinimalisticImagePartOfABook" id="minimalistic_image_part_of_a_book_obj_0" permissions="res-default">
```

File: rosetta.xml [hunk 2]
```diff
@@ -1956,8 +1964,56 @@
...
+        </text-prop>
+        <resptr-prop name="isSequenceOf">
+            <resptr permissions="prop-default">video_0</resptr>
+        </resptr-prop>
+        <interval-prop name="hasSequenceBounds">
+            <interval permissions="prop-default">2.0:4.0</interval>
+        </interval-prop>
+    </resource>
+
+    <resource label="Minimalistic section of an Audio" restype=":MinimalisticSequence" id="minimalistic_sequence_1" permissions="res-default">
```
Comment (irinaschubert) [inline rosetta.xml:2008]: I wouldn't call all these things "minimalistic". As it is an example project it is clear that the examples are minimalistic (or it doesn't really matter if they are or not). For the sake of readability I would leave this term away.

**Review by irinaschubert — APPROVED** [review]
LGTM, thanks!

File: rosetta.xml (hunk 2 above)
Comment (jnussbaum) [reply rosetta.xml]: You're right, it's too complicated. I have now removed the subproperties of `isPartOf`, `seqnum`, `isSequenceOf` and `hasSequenceBounds`. They are now only used directly in the classes of the onto. This 
 - simplifies the onto
 - makes all the "minimalistic" stuff obsolete
 - encourages flat ontologies

**Pull request merged**